web/js/panel.js

```diff
@@ -4013,7 +4013,7 @@ function setBlockColor(blockName, newColor) {
     if (isDefined($blk)) {
         $gBlks[blockName].blockcolor = newColor;
     } else {
-        jmri.log("ERROR: block " + blockName + " not found for color " + newColor);
+        console.error("ERROR: block " + blockName + " not found for color " + newColor);
```

Comment:
Note: You *cannot* rely on the console existing--there is no reason not to expect `console.error` to crash out the JavaScript completely on user's devices, so please revert this, or come up with a reliable alternative.

There is *zero* benefit to writing code to making unit testing easy if if crashes out entirely for users.